### ISSUE TITLE
Chat setting question

### ISSUE THREAD

**Issue by Sonia-js** [fgl27/SmartTwitchTV#148]
Hello. When viewing the chat how do I turn on the badges and icons next to peoples names so it doesn’t look indented. I’ve attached a picture from your app and one from twitch. I’m sure there is a setting somewhere but can’t find it?

![40488CFD-1B20-4D09-9C47-9F4DF2077408](https://github.com/fgl27/SmartTwitchTV/assets/123785838/b9e881f2-9bee-4086-85e0-e03551f7b832)
![D5D42D05-27CC-426F-AB8C-EC60806EA754](https://github.com/fgl27/SmartTwitchTV/assets/123785838/43697570-9e67-489f-9679-63e9fa4a96c8)

**Comment by fgl27** [comment]
twitch changed they badge api, will update later.

there is no way to disable or enable badges.

thanks for the info.

**Comment by fgl27** [comment]
Update it

![image](https://github.com/fgl27/SmartTwitchTV/assets/13755892/f043aee3-b413-4074-90c9-407df14c0253)

**Comment by Sonia-js** [comment]
Yay thank you so much! I can see it is fixed now

**Comment by bitteL3BN** [comment]
I already commented but deleted that comment because I felt I had not done enough troubleshooting until this point and I thought I fixed it but I did not. 

I am on an Nvidia Shield Pro so it shouldn't be a hardware problem I think. And it might have to do with the same changes api that's why I comment here. 
The badges are loading but not how they are supposed to. If I freshly start the app. Watching one stream everything seems fine. But after some time or when I switch channels the majority of the subbadges don't load with the pictures of the different subage tiers. Some of them are just loading as the default stars that someone would have if the streamer didn't modify them.

![20230603_082511](https://github.com/fgl27/SmartTwitchTV/assets/81565486/cda926bc-d820-486e-ab10-cc2ee5418a58)
